frontend/src/pages/MessagingQueues/MQDetails/MQTables/MQTables.tsx

```diff
@@ -35,6 +40,33 @@ export function getColumns(data: any): any[] {
 		title: convertToTitleCase(clm.name),
 		dataIndex: clm.name,
 		key: clm.name,
+		render: [
+			'p99',
+			'error_rate',
+			'throughput',
+			'avg_msg_size',
+			'error_percentage',
+		].includes(clm.name)
+			? (value: number | string): string => {
+					if (!isNumber(value)) return value.toString();
+					return (typeof value === 'string' ? parseFloat(value) : value).toFixed(3);
+			  }
+			: (text: string): ColumnTypeRender<Record<string, unknown>> => ({
+					children:
+						clm.name === 'service_name' ? (
+							<Typography.Link
+								onClick={(e): void => {
+									e.preventDefault();
+									e.stopPropagation();
+									history.push(`/services/${text}`);
```

Comment:
what if the service name contains spaces or special characters ? encode and use the same `encodeURIComponent`

Reply:
done